index.html

```diff
@@ -33,111 +33,108 @@ <h1>Roland<br> Ossisa Yuma</h1>
             </section>
 
             <section class="works">
-                <div class="title set">
-                    <h1>Projects</h1>
+                <div class="grid-item">
+                    <div class="title set">
+                        <h1>Projects</h1>
```

Comment:
- [ ] [OPTIONAL] Kindly try to use only one `h1` in the whole page, this helps immensely with SEO and is generally a good practice. You can use one `h1` tag for the main header of the page and then use `h2`-`h6` depending on the titles hierarchy in the page.